tensorflow/compiler/tf2tensorrt/convert/utils.cc

```diff
@@ -261,13 +261,13 @@ int GetNumberOfEngineInputs(const nvinfer1::ICudaEngine* engine) {
 string ProfileStrategyToName(const ProfileStrategy strategy) {
   switch (strategy) {
     case ProfileStrategy::kRange:
-      return "Range";
+      return "range";
     case ProfileStrategy::kOptimal:
-      return "Optimal";
+      return "optimal";
     case ProfileStrategy::kRangeOptimal:
-      return "Range+Optimal";
+      return "range+optimal";
     case ProfileStrategy::kImplicitBatchModeCompatible:
-      return "ImplicitBatchModeCompatible";
+      return "implicitbatchmodecompatible";
```

Comment:
Why do we want to change this? I though the previous way is more readable...

Reply:
Yes, the previous is more readable. But when we use [this to generate uppercase names](https://github.com/tensorflow/tensorflow/blob/9bc6793a6469be2582f660de575eded21d46df24/tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc#L503), then TRTEngineop constructor cannot use `ProfileStrategyFromName`.

Alterantively `ProfileStrategyFromName` could be changed to lowercase the input string.

Reply:
Yes, let's add a line to ProfileStrategyFromName for this.